app/src/renderer/views/export.pug

```diff
@@ -0,0 +1,15 @@
+extends _extend
+
+block head
+  title Exporting
+  +stylesheet('export')
+  +script('export')
+
+block body
+  +window-header(title='Exporting' updates).is-light.is-draggable
```

Comment:
Why are you not using the native traffic lights? We should use those whenever possible. The reason we don't use them in the editor windows is so that we can show them only on hover.

Reply:
@sindresorhus My preference is using the default traffic lights and even the default title bar... But I thought we're doing it custom since the pug mixin seemed general purpose.